nodejs/awsx/ecs/taskDefinition.ts

```diff
@@ -146,8 +146,8 @@ export abstract class TaskDefinition extends pulumi.ComponentResource {
     // Default policy arns for the Task role.
     public static defaultTaskRolePolicyARNs() {
         return [
-            // Provides wide access to "serverless" services (Dynamo, S3, etc.)
-            aws.iam.ManagedPolicies.AWSLambdaFullAccess,
+            // Provides full access to Lambda
+            aws.iam.ManagedPolicy.LambdaFullAccess,
```

Comment:
Is this in *any* way breaking? i.e. is there any potential issue that someone has already deployed a Task role with the old IAM Policy and then will be downgraded to his Policy?

Reply:
Yes, this is breaking as noted in the description above.

Reply:
Ok, do we need to give an indication of what they need to do if they still need the old role?

Reply:
Good point, I'll add this to the changelog. After I merge this, I'll go ahead and cut a release.